lib/configuration/variables/sources/env.js

```diff
@@ -21,8 +21,8 @@ module.exports = {
       errorCode: 'INVALID_ENV_SOURCE_ADDRESS',
     });
 
-    if (!process.env[address]) missingEnvVariables.add(address);
-    return { value: process.env[address] || null, isPending: !isSourceFulfilled };
+    if (process.env[address] == null) missingEnvVariables.add(address);
+    return { value: process.env[address], isPending: !isSourceFulfilled };
```

Comment:
It should be: `value: process.env[address] == null ? null : process.env[address]`

Reply:
@medikoo fixed